crates/evm/src/evm/system_contracts/src/Bridge.sol

```diff
@@ -11,18 +14,35 @@ import "./L1BlockHashList.sol";
 /// @author Citrea
 
 contract Bridge is MerkleTree, Ownable {
-    // TODO: Update this to be the actual address of the L1BlockHashList contract
-    L1BlockHashList public constant BLOCK_HASH_LIST = L1BlockHashList(address(0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD)); 
+    using BTCUtils for bytes;
+    using BytesLib for bytes;
+
+    struct DepositParams {
+        bytes4 version;
+        bytes2 flag;
+        bytes vin;
+        bytes vout;
+        bytes witness;
+        bytes4 locktime;
+        bytes intermediate_nodes;
+        bytes block_header;
+        uint256 block_height;
+        uint256 index;
+    }
+
+    L1BlockHashList public constant BLOCK_HASH_LIST = L1BlockHashList(address(0x3100000000000000000000000000000000000001)); 
 
-    bytes public DEPOSIT_TXOUT_0 = hex"c2ddf50500000000225120fc6eb6fa4fd4ed1e8519a7edfa171eddcedfbd0e0be49b5e531ef36e7e66eb05"; 
+    bytes public depositScript;
+    bytes public scriptSuffix;
     uint256 public constant DEPOSIT_AMOUNT = 1 ether;
     address public operator;
     mapping(bytes32 => bool) public blockHashes;
-    mapping(bytes32 => bool) public spentTxIds;
+    mapping(bytes32 => bool) public spentWtxIds;
+    uint256 public verifierCount;
```

Comment:
this is Witness script sig input size

we can rename this to requiredSigsCount